test/integration/extension/lib/catalog/product/repository/ShopgateDescriptionRepositoryTest.js

```diff
@@ -1,12 +1,20 @@
 const Credentials = require('../../../../../../../.integration-credentials')
 const BigCommerceFactory = require('../../../../../../../extension/lib/steps/BigCommerceFactory')
 const ShopgateDescriptionRepository = require('../../../../../../../extension/lib/catalog/product/repository/ShopgateDescriptionRepository.js')
+const StoreLogger = require('../../../../../../../extension/lib/store/logger/StoreLogger')
 
 const factory = new BigCommerceFactory(Credentials.clientId, Credentials.accessToken, Credentials.storeHash)
 
 describe('Product description by product id', () => {
+  const context = {
+    log: {
+      info: function(message) {
+
+      }
```

Comment:
to be consistent with the other context objects in the other test cases put the brackets in one line: info: function(message) {}

Reply:
corrected